### PR TITLE
First iteration on Inductiva's Home page

### PR DESCRIPTION
**Pull request by maiyahershey**
This PR addresses documentation, style, text, and changes on Inductiva's Home.

In this iteration, I completely revamped the text, showcased the target audience, the problem and solution inductiva is tackling, and how it works.

The text might have some weaknesses in the technical references, the GIF was an experiment to see how it works and looks, and if adding an infographic brings anything new to the table. 

**Feedback welcome!!**

### CODE REVIEW

File: docs/Home.md [hunk 1]
```diff
@@ -1,18 +1,66 @@
...
+is a beast of a task. As a team of scientists and researchers who have navigated 
+these challenges firsthand, we should know. Scaling these simulations to handle 
+extensive data or complex models typically requires a high level of technical expertise,
+and right now, scientists and researchers are up against a mountain of a learning 
+curve: adapting to increasingly complex systems designed for scaling simulations 
+which often involves managing multiple machines and navigating a maze of technical 
+complexities, tools, cloud environments, and APIs, sidetracking researchers from 
+the cool science they'd rather be doing. 
+
+That’s exactly why we built Inductiva. We've been there, we get it, and we're here
```
[the remark this answers is not in the document]
Comment (maiyahershey) [reply docs/Home.md:25]: agreed!

File: docs/Home.md [hunk 2]
```diff
@@ -1,18 +1,66 @@
...
+to streamline and simplify the simulation process for the scientific community.
+
+## How does it work?
+
+![Inductiva Infographic Animated_GIF (1)](https://github.com/maiyahershey/writeright/assets/102880975/e848088c-182c-4875-acea-ce463b4e4e2f)
+
+The heart of Inductiva lies in its API cluster, which enables scientists to configure, 
+execute, and scale their simulations effortlessly with just two lines of code. This tool 
+allows for running multiple simulations simultaneously and in a distributed manner, 
+whether from the researchers' local computers or via Inductiva's cloud platform.
```
[the remark this answers is not in the document]
Comment (maiyahershey) [reply docs/Home.md:35]: I did get that impression - but could be mistaken! I thought Inductiva gives you that kind of access (MachineGroups)? But I am loving this back and forth, i want to know if objectively i would be able to understand what it does.

File: docs/Home.md [hunk 3]
```diff
@@ -1,18 +1,66 @@
...
+![Inductiva_Logo_noBG(259 x 76 px)](https://github.com/inductiva/inductiva/assets/102880975/c67e31eb-2775-4ee5-ac64-027f4591c0cc)
+# Inductiva: a Python-based software package designed for executing large-scale simulations of physical phenomena, leveraging cluster computing for enhanced scalability and efficiency
+
+Inductiva is an innovative Python package for researchers, scientists, and engineers 
+who need to perform complex simulations like weather patterns, fluid dynamics, or 
+mechanical systems without the constraints of local hardware limitations. It 
+revolutionizes the way simulations of real-world physical phenomena are conducted, 
+addressing a critical challenge in the field.
+
+> _While Inductiva stands out with its capability for cluster computing, it is not
```
[the remark this answers is not in the document]
Comment (maiyahershey) [reply docs/Home.md:10]: most def! priceless insights @IvanPombo!!

File: docs/Home.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (IvanPombo) [reply docs/Home.md:35]: So currently, the API allows to run simulations on the cloud. Either, they will run on a set of machines we have up, which aren't very powerful. Or, users can launch their own dedicated resources (`MachineGroups`), with several machines and select them to run their simulations.

So, technically simulations are all running in the cloud. In their local computers users only have access to the Python client that allows to select and manage this things, but simulations cannot be run on the users computer. 

In the near future, things may change and users may run the simulations in machines they have. But I would avoid going there for now.

[the remark this answers is not in the document]
Comment (maiyahershey) [reply docs/Home.md:35]: got it! i'll give this another go until we get it JUST RIGHT

File: docs/Home.md [hunk 4]
```diff
@@ -1,18 +1,71 @@
...
-and corresponding dependencies.
-- 🚀 Allows running hundreds or even thousands of simulations concurrently, with 
-no coding.
-- 💽 Automatically optimizes hardware configurations for each type of simulation 
-(e.g., CPU vs. GPU, appropriate number of CPU cores, RAM, etc.).
-- 🐍 With Inductiva API, you are not limited to a pre-defined GUI or intricate 
-configuration languages and scripts. Instead, you write small Python programs 
-that seamlessly integrate with your existing codebase and ML framework.
+![Inductiva_Logo_noBG(259 x 76 px)](https://github.com/inductiva/inductiva/assets/102880975/c67e31eb-2775-4ee5-ac64-027f4591c0cc)
+# Inductiva: a Python package for running large-scale simulations of physical phenomena on the Cloud.
```
Comment (sarmento) [inline docs/Home.md:2]: I would go even tighter:

 Inductiva: a Python package for scaling simulations on the Cloud (and on premise?)

File: docs/Home.md [hunk 5]
```diff
@@ -1,18 +1,71 @@
...
+
+Inductiva is a Python package designed for researchers, scientists, and engineers 
+focused on leveraging the full potential of [free and open-source physical simulation software packages]().
```
Comment (sarmento) [inline docs/Home.md:5]: I think the ideas are here, but we may be able to make this even more compact, by dropping words that will become obvious later.

Inductiva is an API that allows scientists and engineers to fully exploit the potential of [open-source simulation software] and scale their simulation projects to hundreds of machines by merely writing a simple python script. 

Inductiva API abstracts way all the complexity of configuring, managing and optimizing costs of Cloud resources, and provides an easy-to-use Python interface to launch a variety of computational resources, including MPI clusters, and orchestrate simulation pipelines that use open-source simulation packages.  

wdyt: @sergiomsantos @hpenedones ?

File: docs/Home.md [hunk 6]
```diff
@@ -1,18 +1,71 @@
...
+you will learn:
+
+* [What is the Inductiva API?]()
+* [What are Inductiva API's core functionalities?]()
+* [What can you do?]()
+
+## What is the Inductiva API?
+
+Inductiva API is a Python package that enhances the usability and scalability of 
+[free and open-source physical simulation software packages](), which are pivotal
```
Comment (sarmento) [inline docs/Home.md:20]: I like this paragraph, but I would refrain from larger claims like "which are pivotal to..."

But I like the rest :)

File: docs/Home.md [hunk 7]
```diff
@@ -1,18 +1,71 @@
...
+* [What are Inductiva API's core functionalities?]()
+* [What can you do?]()
+
+## What is the Inductiva API?
+
+Inductiva API is a Python package that enhances the usability and scalability of 
+[free and open-source physical simulation software packages](), which are pivotal 
+in advancing science and engineering. These software packages, while powerful, often 
+present significant challenges in terms of installation, operation, and particularly, 
+scaling for large-scale computations. This complexity can restrict the potential
```
Comment (sarmento) [inline docs/Home.md:23]: scaling to large computational infrastructures, such as the Cloud or on-premise clusters.

File: docs/Home.md [hunk 8]
```diff
@@ -1,18 +1,71 @@
...
+## What is the Inductiva API?
+
+Inductiva API is a Python package that enhances the usability and scalability of 
+[free and open-source physical simulation software packages](), which are pivotal 
+in advancing science and engineering. These software packages, while powerful, often 
+present significant challenges in terms of installation, operation, and particularly, 
+scaling for large-scale computations. This complexity can restrict the potential 
+of engineers and scientists to fully exploit these tools.
+
+Inductiva addresses these challenges by providing on-demand scaling capabilities
```
Comment (sarmento) [inline docs/Home.md:26]: I think the idea is there, but we need a little change or two.

I am not sure yet that "we provide" scaling. That depends on our business model (are we a "cloud company"?).
What we do is we facilitate scaling. We simplify the task of scaling. 
Again this is subtle, and I will give you more context, but think of us more as a "broker" (man in the middle) than as the executor. The executor -- the one that actually provides computational resources -- can be either an external cloud provider, or even the university computer. We can install there too. So, try to see as as brokers.

File: docs/Home.md [hunk 9]
```diff
@@ -1,18 +1,71 @@
...
+present significant challenges in terms of installation, operation, and particularly, 
+scaling for large-scale computations. This complexity can restrict the potential 
+of engineers and scientists to fully exploit these tools.
+
+Inductiva addresses these challenges by providing on-demand scaling capabilities 
+to high-performance systems, making it simpler for users to deploy these simulators 
+at scale without the necessity of specialized systems engineering or high-performance 
+computing (HPC) team support. 
+
+In essence, **Inductiva's focus lies in computation**: making advanced [computational resources]()
```
Comment (sarmento) [inline docs/Home.md:31]: I think the idea is there again, but we need to refine the style and there is the subtlety that I mentioned before about we not necessarily "making" anything available ourselves. We "handle" existing resources. It's more about  improving the usability of resources that are already there (including open-source software packages) than making anything new available. It's just a subtle difference, but it is important.

File: docs/Home.md [hunk 10]
```diff
@@ -1,18 +1,71 @@
...
+You will interact with these cloud-based simulations through a Python client on 
+your local computer. This interface enables you to select, manage, and scale your 
+simulations in the cloud, leveraging the scalability and power of Inductiva's 
+[computational resources](), while bypassing the limitations of your local hardware.
+
+This API offers a range of benefits for diverse simulation needs:
+
+|  	|  	|
+|---	|---	|
+| Unified Platform 	| _It unifies multiple simulation packages on a single platform, covering various <br>multi-physics aspects that can encompass domains like fluid dynamics, molecular dynamics, <br>plasmas, and structural mechanics. This integration allows for a streamlined, user-friendly <br>access point, eliminating the need to manage separate software for each type of simulation._ 	|
```
Comment (sarmento) [inline docs/Home.md:53]: It unifies multiple standard open-source simulation packages...

File: docs/Home.md [hunk 11]
```diff
@@ -1,18 +1,71 @@
...
+| Hassle-Free Setup 	| _It eliminates the need for installing and managing <br>complex simulation software and corresponding dependencies_ 	|
+| Python-Powered Flexibility 	| _It empowers you to write simple <br>Python scripts that blend seamlessly with your existing codebase and machine learning <br>frameworks, unlocking a world of customization and control._ 	|
```
Comment (sarmento) [inline docs/Home.md:55]: Not sure about " unlocking a world of customization and control" @hpenedones ?

File: docs/Home.md [hunk 12]
```diff
@@ -1,18 +1,71 @@
...
+| Smart Hardware Optimization 	| _It automatically tunes hardware configurations – <br>whether you need CPU or GPU, decides the right number of cores, RAM, etc., specific <br>to each simulation type._ 	|
```
Comment (sarmento) [inline docs/Home.md:56]: I think it's more general than that, and in fact, this seems it is going one step too far.
I like talking about optimization but:

1) just the orchestration itself -- even if it is not optimized -- is a great feature. Setting up an MPI cluster is a huge thing. But also other things that we already have like auto-scaling of resources, and automatic termination of resources are super super important. I would prefer to talk more about this @hpenedones @sergiomsantos ?

2) We also need to make a reference to the cost and cost visibility @sergiomsantos ?

**Review by sarmento — COMMENTED** [review]
Maya: I think this was another great step and we are now moving into talking about more subtle things. This is actually great :)

I left a few more comments.

**Review by sarmento — APPROVED** [review]
Let's merge and then iterate on the result.

**Pull request merged**